scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala

```diff
@@ -148,21 +157,36 @@ object ScaladocParser {
     }
   }
 
-  private def listBlockParser[_: P](minIndent: Int = 1): P[ListBlock] = P {
+  private def listItemParser[_: P](indent: Int): P[ListItem] = {
+    def innerBody =
+      (listBlockParser(indent) |
+        mdCodeBlockParser(indent) |
```

Comment:
here's what i found:
- while nested list can be indented by just 1 space, a nested markdown block has to be indented by the width of the list marker *and* the space that follows it, so we need two parameters here
- we can also include a nested `{{{` code block, it just doesn't need to have any special indentation.

Reply:
good points!

Do you have a fix for that or should I do that after we merge your PRs?

Reply:
i do have a fix, will share in a couple of hours. expanded your solution, added embedded `{{{` code blocks, also option for tags (someone complained that `@example` tag needs to have embedded code).

Reply:
shared part 1 (of 3): #2537. there are two more commits. let me know if you prefer to see everything at once, rather than one piece of functionality at a time, and i will upload more commits into same pr. otherwise, will submit more after review and merge.